<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Application</class>
 <widget class="QMainWindow" name="Application">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="FileUpload" name="uploadButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>186</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QToolButton#upload_button{
	 background-color: none;
    border-style: dashed ;
    border-width: 2px;
    border-radius: 0px;
    border-color: #c4c4c4;
    min-width: 10em;
    padding: 6px;
}

QToolButton#upload_button:hover {
    background-color: #e5e5e5;
}
</string>
      </property>
      <property name="text">
       <string>Upload</string>
      </property>
      <property name="icon">
       <iconset>
        <normalon>:/app/image/file-upload-solid.svg</normalon>
       </iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="uploadProgress">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="processEntries"/>
    </item>
    <item>
     <widget class="QPushButton" name="clearBtn">
      <property name="text">
       <string>clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileUpload</class>
   <extends>QToolButton</extends>
   <header location="global">app/FileUpload.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
